sdk/storage/storage-file-share/src/ShareDirectoryClient.ts

```diff
@@ -1346,9 +1347,10 @@ export class ShareDirectoryClient extends StorageClient {
         );
         marker = response.marker;
         response.closedHandlesCount && (handlesClosed += response.closedHandlesCount);
+        response.numberOfHandlesFailedToClose && (numberOfHandlesFailedToClose += response.numberOfHandlesFailedToClose);
```

Comment:
For consistency this should be renamed to `closeFailureCount`?

Reply:
`closeFailureCount` seems to be a little bit obsure about what it counts. I will use it if no better name comes up.